module_utils/vyos/facts/interfaces/interfaces.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The vyos interfaces fact class
+It is in this file the configuration is collected from the device
+for a given resource, parsed, and the facts tree is populated
+based on the configuration.
+"""
+from copy import deepcopy
+from ansible.module_utils. \
+     vyos.facts.base import FactsBase
+from re import findall, M
+import q
```

Comment:
delete this line, and test should pass. We'll make sure to bootstrap the project properly moving forward

Reply:
Fixed